ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -154,7 +157,18 @@ const AccountListItem = ({
         ? account.balance
         : totalFiatBalance;
   } else {
-    balanceToTranslate = accountTotalFiatBalances.totalBalance;
+    const balanceOrFallback = accountTotalFiatBalances?.totalBalance ?? 0;
+    const bnBalance = new BigNumber(balanceOrFallback);
+    const formattedBalanceToTranslate = formatWithThreshold(
+      bnBalance.toNumber(),
+      0.00001,
```

Comment:
What does this represent?

Reply:
It's the lower threshold. Meaning, that if the balance is less than `0.00001` we append a `<` to the threshold and display it.

So if the balance is `0.00000001` we will display `<0.00001`

This is partially relevant to the ADR we discussed in Slack.